docs/reference/system/roles.md

```diff
@@ -156,35 +174,64 @@ query {
 }
 ```
 
----
+</template>
+</SnippetToggler>
 
 ## Create a Role
 
 Create a new role.
 
-### Query Parameters
+### Request
+
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
+
+`POST /roles`
+
+```json
+{
+	"role_object_field_1": "value_1",
+	"role_object_field_2": "value_2",
+	"role_object_field_3": value_3,
+	"role_object_field_4": value_4,
+	"role_object_field_5": value_5,
```

Comment:
What's the idea here to use `value_` instead of `"value_"`?

Reply:
An error.